tests/fixture/tmpnet/network.go

```diff
@@ -253,6 +253,11 @@ func (n *Network) EnsureDefaultConfig(log logging.Logger) error {
 		}
 	}
 
+	emptyRuntime := NodeRuntimeConfig{}
+	if n.DefaultRuntimeConfig == emptyRuntime {
+		return errMissingRuntimeConfig
```

Comment:
nit we could just return the error without a sentinel error since we don't use it (i.e. with errors.Is)

```suggestion
		return errors.New("DefaultRuntimeConfig must not be empty")
```